docs/hugo/content/introduction/installing-from-yaml.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Installation: From YAML"
+---
+
+### Prerequisites
+1. You have followed the [installation instructions](https://azure.github.io/azure-service-operator/#installation) up to the "install from Helm" step.
+
+### Installation
+
+1. Install [the latest **v2+** release](https://github.com/Azure/azure-service-operator/releases) of Azure Service Operator.
+   ```bash
+   kubectl apply --server-side=true -f https://github.com/Azure/azure-service-operator/releases/download/v2.0.0-alpha.6/azureserviceoperator_v2.0.0-alpha.6.yaml
+   ```
+2. Create the Azure Service Operator v2 secret. This secret contains the identity that Azure Service Operator will run as. Make sure that you have the 4 environment variables from step 2 set before running this command.
```

Comment:
This _is_ Step 2 - so "... environment variables from step 2 set ..." is confusing to me.

Is this referring back to the installation instructions listed as a prerequisite? If so, perhaps just list having those variables set as a second pre-requ.

Reply:
Fixed. Hopefully should be clearer now.